### PR TITLE
Add Content ID to Contentful Adapter link parser

### PR DESCRIPTION
**Pull request by altruity**
https://lastrev.atlassian.net/browse/INTERNAL-493

### CODE REVIEW

File: packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ export default ({
     case manualEntryTypeText:
       if (!manualUrl) {
         warn(
-          `Bad content for ${contentTypeId}: DestinationType is ${manualEntryTypeText}, but no URL has been entered`
+          `Bad content for ${contentTypeId} with Content ID ${contentReference?.sys.id}: DestinationType is ${manualEntryTypeText}, but no URL has been entered`
```
Comment (prceasar) [inline packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts:50]: contentReference is wrong here, what we want is the ID of the ElementLink item, which is the one that has the missing fields. contentReference is a field within this item that may or may not have an item in it.  If the ID of the ElementLink item is not available in LinkParser, we may have to add another argument and pass it in from the caller.

**Review by prceasar — CHANGES_REQUESTED** [review]

File: packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts (hunk 1 above)
Comment (altruity) [reply packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts:50]: @prceasar Gotcha, thx! Just curious then, is that ID the same `id` listed here?
https://github.com/last-rev-llc/lastrev-toolkit/blob/ef881b72502bb75a9869aba3ee64ae01dedd8925/packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts#L31
..which is passed into the caller here:
https://github.com/last-rev-llc/lastrev-toolkit/blob/98a1c9f3c44cf4ace22e969a60872fb9db315a8a/packages/lastrev-adapter-contentful/src/contentfulAdapter/contentfulAdapter.ts#L41

Comment (prceasar) [reply packages/lastrev-adapter-contentful/src/linkParser/linkParser.ts:50]: @altruity Yes, I believe that's it.

**Review by prceasar — APPROVED** [review]
Looks good! If you want to just verify that the messages work correctly, you can run this command `yarn build:external {relative path to project, such as impossiblefoods.com}` and then running the prefetch on impossible foods `npx lr-prefetch`. Once verified, or if you already have, feel free to merge. Once merged, I will publish the changes.

**Comment by altruity** [comment]
@prceasar Oh gosh, somehow this msg got buried in my inbox. 
I attempted to verify, but couldn't get the ID to show up in the log.
That said, seems like it should work ok. Sorry for the late reply. 😊

**Pull request merged**